### ISSUE TITLE
Instagram Story like feature

### ISSUE THREAD

**Issue by adw0rd** [adw0rd/instagrapi#790]
### Discussed in https://github.com/adw0rd/instagrapi/discussions/788

<div type='discussions-op-text'>

<sup>Originally posted by **rdxshubham** July 16, 2022</sup>
Do we have a function to like a story (a new feature rolled out a few months back by Instagram, a small heart icon in the story)?  </div>

**Comment by adw0rd** [comment]
instagrapi 1.16.22

**Comment by rdxshubham** [comment]
Thanks, @adw0rd for the implementation. It is working perfectly fine.

**Comment by adw0rd** [comment]
@bilakos26 refresh your instagram app

**Comment by bilakos26** [comment]
> @bilakos26 refresh your instagram app

you are right, with deleting and reinstalling the app it showed the likes.
Great work once again.
